ui/src/components/style/CommonLayoutStyle.js

```diff
@@ -21,19 +21,13 @@ export const CenteredPageContainer = styled.div`
 `;
 
 export const LeftSideInstanceList = styled.div`
-  flex-direction: column;
-  min-height: 696px;
-  width: 49%;
-  margin-bottom: 20px;
-  min-width: 49%;
+  flex: 1;
```

Comment:
Got rid of the nested component which gave `flex: 1` in core-ui and thus had to use it here. The reason I got rid of it is because I needed to pass some extra props, but the extra (`rest`) params were passed to the FillAvailableFlexBox component. As I warned earlier - it's bad design practice to go along this "one-size-fits-all" approach, which gives developer convenience for some use-cases at the expense of preventing further customization